erp/imports/serializers.py

```diff
@@ -208,14 +208,27 @@ def validate(self, obj):
 
         existing = Erp.objects.find_duplicate(
             numero=obj.get("numero"),
-            voie=obj.get("voie"),
-            lieu_dit=obj.get("lieu_dit"),
             commune=obj["commune"],
             activite=obj["activite"],
+            voie=obj.get("voie"),
```

Comment:
We could revert change if not mandatory

Reply:
(not mandatory)

Reply:
it was more to match the function signature.